deployer/debug.py

```diff
@@ -259,5 +260,167 @@ def start_docker_proxy(
         subprocess.check_call(cmd)
 
 
+@app.command()
+def transfer_old_home_dir_to_new_location(
```

Comment:
Transfer implies `mv` to me, or `copy` + `rm`, but its a plain `copy` operation. I'd appreciate this named with `copy` over `transfer`.

There is an assumption in the command of copying files from one homedir _into a subdir of_ another another users homedir, this also isn't obvious from either naming proposals either.

With this considered, what naming do I think makes sense? I propose `copy_homedir_into_another` as that seems quite short and self descriptive.

Reply:
Agree that the original name I gave it was long and confusing. Thanks for the great suggestions @consideRatio and @yuvipanda. I will go with @consideRatio's proposed name which uses the "copy" instead of the "transfer" verb as that is more descriptive of what the command does.

@yuvipanda, please let me know if you disagree and we can all brainstorm to get to a common proposal